package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "crontab.backend",
+  "version": "1.0.0",
+  "description": "Backend for crontab.training.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yryuvraj/crontab.backend.git"
```

Comment:
Can we update this to use the osdc URL instead of your fork? (and the ones bellow as well)